hyperjumptech/monika
# Fix: 546 stuck after finish all repetition

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
Monika is stuck at doing nothing after finishing all probe repetition

## How did you implement / how did you fix it  
by sending interruption signal to current process after finish all repetition

## How to test  
1. run application with 1 repetition and 0 stun `npm start -- -r 1 -s 0`
2. make sure the application is exited after running probe 1 time.

![Screen Shot 2021-12-22 at 14 43 46](https://user-images.githubusercontent.com/8952493/147054785-3c4940da-3e54-4da1-ae4f-14cbc17f9332.png)
